File: divisores.py
def calculo():
  num = int(input("introducir numero "))
  f = open('divisores.txt','a')
  print("divisores de ",num)
  f.write("\ndivisores de " + str(num))
  for divisor in range(1,num+1):
      if (num % divisor) == 0 :
          print(divisor)
          f.write("\n" + str(divisor))
  f.close()
         
calculo()

File: divisores.m
  num = input("introducir numero ")
  fprintf("divisores de %d",num)
  fid = fopen('divisores.txt','a');
  fprintf(fid,"\ndivisores de %d: ",num)
  for divisor=1:num+1
      m = mod(num,divisor)
      if (m == 0) 
            disp(divisor)
              fprintf(fid,"%d, ",divisor)          
      end
  endfor  
  fclose(fid)
  